source/includes/steps-fle-convert-to-a-remote-master-key-aws.yaml

```diff
@@ -72,6 +108,12 @@ content: |
           - Yes
           - Contains the authentication credentials of the account user
```

Comment:
[suggestion] minor, but I suggest adding periods to the end of the first two descriptions to match the Session Token description you added (or removing the period from the Session Token description, whichever way you want to standardize)